src/components/policy/policy_regular/src/sql_pt_queries.cc

```diff
@@ -233,42 +233,42 @@ const std::string kCreateSchema =
     "    REFERENCES `language`(`code`) "
     "); "
     "CREATE INDEX IF NOT EXISTS `app_level.fk_app_levels_application1_idx` "
-    "  ON `app_level`(`application_id`); "
+    "  ON `app_level`(`application_id` COLLATE NOCASE); "
     "CREATE INDEX IF NOT EXISTS `app_level.fk_app_level_language1_idx` "
     "  ON `app_level`(`app_registration_language_gui`); "
     "CREATE INDEX IF NOT EXISTS `app_level.fk_app_level_language2_idx` "
     "  ON `app_level`(`app_registration_language_vui`); "
     "CREATE TABLE IF NOT EXISTS `nickname`( "
-    "  `name` VARCHAR(100) NOT NULL, "
-    "  `application_id` VARCHAR(45) NOT NULL, "
+    "  `name` VARCHAR(100) NOT NULL COLLATE NOCASE, "
+    "  `application_id` VARCHAR(45) NOT NULL COLLATE NOCASE, "
     "  PRIMARY KEY(`name`,`application_id`), "
     "  CONSTRAINT `fk_nickname_application1` "
     "    FOREIGN KEY(`application_id`) "
     "    REFERENCES `application`(`id`) "
     "); "
     "CREATE INDEX IF NOT EXISTS `nickname.fk_nickname_application1_idx` "
-    "  ON `nickname`(`application_id`); "
+    "  ON `nickname`(`application_id` COLLATE NOCASE); "
     "CREATE TABLE IF NOT EXISTS `app_type`( "
     "  `name` VARCHAR(50) NOT NULL, "
-    "  `application_id` VARCHAR(45) NOT NULL, "
+    "  `application_id` VARCHAR(45) NOT NULL COLLATE NOCASE, "
     "  PRIMARY KEY(`name`,`application_id`), "
     "  CONSTRAINT `fk_app_type_application1` "
     "    FOREIGN KEY(`application_id`) "
     "    REFERENCES `application`(`id`) "
     "); "
     "CREATE TABLE IF NOT EXISTS `request_type`( "
     "  `request_type` VARCHAR(50) NOT NULL, "
-    "  `application_id` VARCHAR(45) NOT NULL, "
+    "  `application_id` VARCHAR(45) NOT NULL COLLATE NOCASE, "
     "  PRIMARY KEY(`request_type`,`application_id`), "
     "  CONSTRAINT `fk_app_type_application1` "
     "    FOREIGN KEY(`application_id`) "
     "    REFERENCES `application`(`id`) "
     "); "
     "CREATE INDEX IF NOT EXISTS `app_type.fk_app_type_application1_idx` "
-    "  ON `app_type`(`application_id`); "
+    "  ON `app_type`(`application_id` COLLATE NOCASE); "
```

Comment:
@LuxoftAKutsan it's supported on QNX
From [qnx.com](http://www.qnx.com/developers/docs/6.5.0/index.jsp?topic=%2Fcom.qnx.doc.qdb_en_dev_guide%2Fdatatypes.html):

> User-defined collation sequences
> 
> By default, when QDB compares two text values, the result of the comparison is determined using memcmp(), regardless of the encoding of the string. QDB lets you supply arbitrary comparison functions, known as user-defined collation sequences, to be used instead of memcmp(). See the chapter Writing User-Defined Functions for more information.
> 
> Aside from the default collation sequence BINARY, implemented using memcmp(), QDB features two extra built-in collation sequences intended for testing purposes, NOCASE and REVERSE:
> 
>     BINARY — Compare string data using memcmp(), regardless of text encoding.
>     REVERSE — Collate in the reverse order to BINARY.
>     NOCASE — The same as BINARY, except the 26 upper-case characters used by the English language are converted to their lower-case equivalents before the comparison is performed.
>